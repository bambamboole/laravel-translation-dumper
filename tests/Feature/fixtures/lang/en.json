{
    "This is a non dotted translation key": "This is a non dotted translation key"
}
