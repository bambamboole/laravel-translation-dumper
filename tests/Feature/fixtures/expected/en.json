{
    "test undotted key": "x-test undotted key",
    "This is a non dotted translation key": "This is a non dotted translation key",
    "word": "x-word",
    "Works.": "x-Works."
}
